hello-world/Makefile

```diff
@@ -1,8 +1,8 @@
 # Contains some helper utilities for updating projects
 
 update-hello: ## Generates a new hello project from rust, builds, and copies the compiled file to rust/build
-	cosmo new actor --path rust hello
```

Comment:
I think we want to keep this as `cosmo` right? Totally fine to use `wash` in our build pipelines, but locally they will be using cosmo since this are cosmonic specific demos